.github/action-security-config.json

```diff
@@ -0,0 +1,57 @@
+[
+  {
+    "name": "github_access_token",
+    "validationRegex": "/^[\\w.\\-]+$/.source",
+    "validationMessage": "Letters, numbers, or .-_ only"
+  },
+  {
+    "name": "pypi_token",
+    "validationRegex": "/^[\\w.\\-]+$/.source",
+    "validationMessage": "Letters, numbers, or .-_ only"
```

Comment:
I need to see what the possible chars are for all these auth fields and make the `validationRegex` and `validationMessage` more specific accordingly.